tests/cypress/tests/standalone/edit_cell_test.ts

```diff
@@ -206,7 +206,58 @@ describe(`edit, mode=${AppMode.Typed}`, () => {
                 );
             });
         });
+
+    });
+});
+
+describe.only(`edit headers, mode=${AppMode.Typed}`, () => {
```

Comment:
I think a file for header tests is warranted.
Don't forget to remove the `.only` :)